testnets/zenrocktestnet/chain.json

```diff
@@ -0,0 +1,129 @@
+{
+    "$schema": "../../chain.schema.json",
+    "chain_name": "zenrocktestnet",
+    "status": "live",
+    "network_type": "testnet",
+    "pretty_name": "Zenrock Testnet",
+    "chain_type": "cosmos",
+    "chain_id": "gardia-2",
+    "bech32_prefix": "rock",
+    "daemon_name": "zenrockd",
+    "node_home": "$HOME/.zenrockd",
+    "key_algos": [
+      "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+      "fee_tokens": [
+        {
+          "denom": "urock",
+          "fixed_min_gas_price": 0.005,
+          "low_gas_price": 0.01,
+          "average_gas_price": 0.025,
+          "high_gas_price": 0.03
+        }
+      ]
+    },
+    "staking": {
+      "staking_tokens": [
+        {
+          "denom": "urock"
+        }
+      ]
+    },
+    "codebase": {
+      "git_repo": "https://github.com/zenrock-foundation/zrchain",
+      "recommended_version": "5.3.4",
+      "compatible_versions": [
+        "5.3.4"
+      ],
+      "cosmos_sdk_version": "0.50",
+      "consensus": {
+        "type": "cometbft",
+        "version": "0.38"
+      },
+      "cosmwasm_version": "0.53",
+      "cosmwasm_enabled": true,
+      "genesis": {
+        "genesis_url": "https://rpc.gardia.zenrocklabs.io/genesis"
+      },
+      "versions": [
+        {
+          "name": "5.3.4",
+          "recommended_version": "5.3.4",
+          "compatible_versions": [
+            "5.3.4"
+          ],
+          "cosmos_sdk_version": "0.50",
+          "consensus": {
+            "type": "cometbft",
+            "version": "0.38"
+          },
+          "cosmwasm_version": "0.53",
```

Comment:
cosmwasm_version, cosmwasm_enabled, and cosmo_sdk_version properties have been deprecated in favor of their repacements, which you have.